docs/axes/radial/linear.md

```diff
@@ -104,7 +104,7 @@ The following options are used to configure the point labels that are shown on t
 | Name | Type | Default | Description
 | -----| ---- | --------| -----------
 | `callback` | `Function` | | Callback function to transform data labels to point labels. The default implementation simply returns the current string.
-| `fontColor` | `Color` | `'#666'` | Font color for point labels.
+| `fontColor` | ``Color/Color[]`` | `'#666'` | Font color for point labels.
```

Comment:
[nit] looks like the backticks got duplicated on each side

Reply:
Sorry about that, just pushed a fix up